flycheck.el

```diff
@@ -8653,6 +8653,15 @@ the `--severity' option to Perl Critic."
   :safe #'integerp
   :package-version '(flycheck . "0.18"))
 
+(flycheck-def-option-var flycheck-perlcritic-theme nil perl-perlcritic
+  "The theme expression for Perl Critic.
+
+The value of this variable is passed as the `--theme' option to `Perl::Critic'.
+See the documentation of `Perl::Critic' for details"
```

Comment:
I think `' quotes are typically just used for ELisp symbols

Reply:
Dunno, I followed what was done in the documentation of flycheck-perlcritic-severity. 
No problem changing it, obviously

Reply:
BTW, just signed the CLA, sorry for being late on it